pipelines/src/pipelines/trigger/__main__.py

```diff
@@ -0,0 +1,5 @@
+from .main import sandbox_run
```

Comment:
this allows you to invoke a python module with `python -m pipelines.trigger` instead of `python -m pipelines.trigger.main`. doesn't provide much value in this case though.